### PR TITLE
Fix sort such that it sorts [Aa - Zz]

### LINKED ISSUE #96: Sort Command: sorts [A-Z] [a-z] instead of [Aa - Zz]
Sort command currently prioritizes uppercase letters over lowercase letters.
This can be solved by doing `String::toLowerCase` in the comparator.

### PR DESCRIPTION
**Pull request by highorbit25**
Resolves #96 
- Fix by using `String::toLowerCase` in the comparator.
- Added `Assignment` and `Participation` as valid attributes to sort by.
- Updates to UG for `sort`

### CODE REVIEW

File: src/main/java/seedu/studmap/model/student/Student.java [hunk 1]
```diff
@@ -59,39 +59,39 @@ public StudentID getId() {
     }
 
     public String getIdString() {
-        return id.toString();
+        return id.toString().toLowerCase();
```
Comment (comicalromance) [inline src/main/java/seedu/studmap/model/student/Student.java:62]: hmm, might not be the best idea to modify the getIdString (and other attribute getters) directly, as there might be other methods / classes that use these getters.
maybe it's possible to modify the comparator instead?
image

**Review by comicalromance — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/studmap/model/student/Student.java (hunk 1 above)
Comment (Piyotato) [reply src/main/java/seedu/studmap/model/student/Student.java:62]: My bad, I suggested a poorer solution without knowing about `String::compareToIgnoreCase`. Use this solution instead?

Comment (highorbit25) [reply src/main/java/seedu/studmap/model/student/Student.java]: Implemented accordingly using String::compareToIgnoreCase as second argument for comparator.

**Review by Piyotato — APPROVED** [review]
LGTM

**Review by comicalromance — APPROVED** [review]
LGTM!

**Pull request merged**